src/interfaces/BookingPlaceData.ts

```diff
@@ -5,6 +5,7 @@ interface BookingPlaceData {
   startDate: Timestamp
   endDate: Timestamp | null
   visitorName: string
+  visitorId: string
```

Comment:
Let's change it to the object
```suggestion
  visitor: {
    name: string
    id: string
  }
```